src/utils.js

```diff
@@ -120,20 +122,48 @@ export function getVariable(context, name) {
   return result;
 }
 
+export function getLocale() {
+  return osLocale.sync();
+}
+
+export function getLocaleDir(locale) {
+  return `./locale/${locale}/messages.js`;
+}
+
+export const lib = {
+  getLocaleDir,
+};
```

Comment:
This is introduced because of the approach mentioned here : http://luetkemj.github.io/170421/mocking-modules-in-jest/